cpp/src/arrow/engine/substrait/extension_set.cc

```diff
@@ -954,7 +954,9 @@ ExtensionIdRegistry::SubstraitAggregateToArrow DecodeBasicAggregate(
         return Status::Invalid("Expected aggregate call ", call.id().uri, "#",
                                call.id().name, " to have at least one argument");
       }
-      case 1: {
+      default: {
```

Comment:
This is so that we can support UDFs which can have arity > 1

Reply:
Yes